### PR TITLE
Headline page branch

### PR DESCRIPTION
**Pull request by SamuelTakyiBoateng**
add index.html (toolbar and headline of the first section add)
Style.css ( add style to the toolbar and headline)
ReadMe (Description about project)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <h2 class="header__logo">My Logo</h2>
```
Comment (joskalenda) [inline index.html:12]: - [ ] Well done @SamuelTakyiBoateng, I appreciate your work, however, the logo name you respect is the following:
- It must be an `a` tag to help the user navigate to the home page, Going "back to home" is a very common interaction for users browsing a corporate website
- I believe the `my logo` name on the figma design is just a reference to tell you, please replace this with your initials or your name even, such as the home logo of Facebook is not `my company`, it is Facebook, this to say, please use your name, your logo or your initial letters to represent the logo

File: style.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+/* Headline section */
+.headline {
+  background-image: url(images/aboutMe\ background.png);
+  background-repeat: no-repeat;
```
Comment (joskalenda) [inline style.css:39]: - [ ] Your background image is hiding the text, please fix this.

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @SamuelTakyiBoateng , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Good design
- [ ].. ✅ Green linters
- [ ].... ✅ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (SamuelTakyiBoateng) [reply index.html]: Thank you so much careful consideration and suggestion.
The prescribed changes have been made. Thanks again!

File: style.css (hunk 2 above)
Comment (SamuelTakyiBoateng) [reply style.css:39]: The background has been modified for it not to obstruct the text.
Thank🙏!

**Review by Strangeal — APPROVED** [review]
Hi @SamuelTakyiBoateng, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights
Used GitHub flow ✅
Used linters ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**